app/lib/features/news/widgets/news_item_slide/image_slide.dart

```diff
@@ -40,9 +44,11 @@ class _ImageSlideState extends State<ImageSlide> {
       builder: (BuildContext context, AsyncSnapshot<Uint8List> snapshot) {
         if (snapshot.hasData) {
           return Container(
+            color: widget.bgColor,
+            alignment: Alignment.center,
             foregroundDecoration: BoxDecoration(
               image: DecorationImage(
-                fit: BoxFit.cover,
+                fit: BoxFit.fitWidth,
```

Comment:
okay, now I need this to be explained, please ...

Reply:
I will make to `BoxFit.contain` as `BoxFit.cover` is scaling entire horizontal image. As per our last discussion, we need to remove auto-scaling of the image slide.

![Screenshot 2024-02-07 at 5 34 41 PM](https://github.com/acterglobal/a3/assets/51526480/91bc83fe-4246-481a-a45b-f11056204251)